include/squirrel.h

```diff
@@ -22,6 +22,8 @@ THE SOFTWARE.
 #ifndef _SQUIRREL_H_
 #define _SQUIRREL_H_
 
+#include "squirrel_Export.h"
+

```

Comment:
I think I would prefer using ``-include`` (gcc/clang) and ``/FI`` (Microsoft/Intel) switches here, so that it does not have to be re-done on every Squirrel update.